src/static/helpers/slasHelper.ts

```diff
@@ -104,6 +109,9 @@ export const generateCodeChallenge = async (
  * @param parameters.redirectURI - the location the client will be returned to after successful login with 3rd party IDP. Must be registered in SLAS.
  * @param parameters.hint? - optional string to hint at a particular IDP. Guest sessions are created by setting this to 'guest'
  * @param parameters.usid? - optional saved SLAS user id to link the new session to a previous session
+ * @param privateClient? - flag to indicate if the client is private or not. Defaults to false.
+ * @param options - an object containing the options for this function.
```

Comment:
```suggestion
 * @param options? - an object containing the options for this function.
```

`options` is optional

Reply:
the name options is already an indicator that it is optional. I don't think we need to do that

Reply:
It's minor, but I'm pushing back that we should mark it as optional with `?` to maintain consistency in this file as all other optional arguments are marked with `?` in the TS/JSDocs

I'm also ok updating the SDK to use the square bracket notation to indicate optionality, but we should be consistent in the convention that we use